datanode/sqlstore/migrations/0034_oracle_data.sql

```diff
@@ -0,0 +1,69 @@
+-- +goose Up
+-- First make oracle_data a hypertable
+-- +goose StatementBegin
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT * FROM timescaledb_information.hypertables WHERE hypertable_name = 'stop_orders') THEN
```

Comment:
You're checking for the stop_orders hypertable here, I'm not sure this is correct.

Reply:
tbh I wasn't sure what that was for but figured you had had some fun experience to lead you to put it there..